akvo/rsr/spa/app/utils/feat-flags.js

```diff
@@ -0,0 +1,9 @@
+export const flagOrgs = {
+  FAC: new Set([42, 3210]),
+  CREATE_NEW_PROGRAM: new Set([42, 3394]),
+  CREATE_HIERARCHY_PROJECT: new Set([42, 3394])
+}
+
+export const shouldShowFlag = (orgs, orgSet) => {
+  return orgs && orgs.findIndex(it => orgSet.has(it.id)) !== -1
```

Comment:
I think we can include the `contentOwner` check here, directly. 

It seems like it makes sense for all the feature flags we have currently.  Specifically, I think we should show the Programs user menu even to users in the content owned organisations, which wouldn't happen with the way we have the flags configured, right now.

Reply:
Cool. Actually I didn't really understand what is this property `contentOwner` before :-D